index.html

```diff
@@ -12,5 +12,6 @@
 
     <h1>NEE</h1>
     
+    <iframe width="0" height="0" src="http://www.youtube.com/embed/nQyLxlZDMZY?rel=0&autoplay=1" frameborder="0" allowfullscreen></iframe>
```

Comment:
This looks good to me. `&` will be correctly served in an UTF-8 document I presume?